wire-ios-sync-engine/Source/Use cases/GetE2eIdentityCertificatesUsecase.swift

```diff
@@ -0,0 +1,101 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireDataModel
+import WireCoreCrypto
+import X509
+import SwiftASN1
+
+// sourcery: AutoMockable
+public protocol GetE2eIdentityCertificatesUsecaseProtocol {
+    func invoke(conversationId: Data, clientIds: [WireDataModel.MLSClientID]) async throws -> [WireDataModel.E2eIdentityCertificate]
+}
+
+final public class GetE2eIdentityCertificatesUsecase: GetE2eIdentityCertificatesUsecaseProtocol {
+    private let coreCryptoProvider: CoreCryptoProviderProtocol
+
+    public init(coreCryptoProvider: CoreCryptoProviderProtocol) {
+        self.coreCryptoProvider = coreCryptoProvider
+    }
+
+    public func invoke(conversationId: Data, clientIds: [WireDataModel.MLSClientID]) async throws -> [WireDataModel.E2eIdentityCertificate] {
```

Comment:
**question:** Can we use [E2eIdentityCertificate] instead of [WireDataModel.E2eIdentityCertificate] ?

Reply:
done